plugins/modules/dcnm_network.py

```diff
@@ -83,6 +83,11 @@
         type: int
         required: false
         note: If not specified in the playbook, DCNM will auto-select an available vlan_id
+      routing_tag:
+        description: 'Routing Tag for the network profile'
+        type: int
+        required: false
```

Comment:
Please add a default key for routing_tag in the doc.

```default: 12345```

Reply:
With this update you can remove the note because it's implied that 12345 will be used if not specified explicitly in the playbook

Reply:
done.